### PR TITLE
Move test-infra IPv6 job to use pytest workflow

### PR DESCRIPTION
**Pull request by eliorerz**

### CODE REVIEW

File: ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml [hunk 1]
```diff
@@ -186,9 +173,7 @@ tests:
     env:
       ASSISTED_CONFIG: |
         NUM_MASTERS=1
-        NUM_WORKERS=0
-        ISO_IMAGE_TYPE=minimal-iso
-        OPENSHIFT_VERSION=4.8
+        MAKEFILE_TARGET="create_full_environment run test_parallel"
     workflow: baremetalds-assisted-install
 - as: e2e-metal-assisted-single-node-periodic
```
Comment (YuviGold) [inline ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml:178]: Change this periodic as well

**Comment by YuviGold** [comment]
Please change the `e2e-metal-assisted-ipv6` job on `assisted-installer`, `assisted-installer-agent` and `assisted-service` repositories as well.

**Comment by YuviGold** [comment]
/lgtm

ocm might fail
/hold

File: ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml [hunk 2]
```diff
@@ -106,33 +106,20 @@ tests:
...
   cron: '@daily'
-  steps:
-    cluster_profile: packet
-    env:
-      ASSISTED_CONFIG: |
-        IPv6=yes
-        IPv4=no
-        VIP_DHCP_ALLOCATION=no
-        PROXY=yes
-    workflow: baremetalds-assisted
```
Comment (omertuc) [inline ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml:125]: Can you please write the reason this job was removed in the commit message?

Comment (YuviGold) [reply ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml:125]: It was a temporary job. But I agree it should be mentioned in the commit message @eliorerz

Comment (eliorerz) [reply ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml:125]: Sure

**Comment by YuviGold** [comment]
`assisted-installer-agent/ocm-2.3/e2e-metal-assisted-ipv6` is a known bug @lranjbar is taking care of.
/lgtm
/unhold

**Pull request merged**